_posts/2019-02-18-event-store-5.0.0-release.md

```diff
@@ -0,0 +1,193 @@
+---
+title: "Event Store 5.0.0"
+author: "Shaan Nobee"
+layout: blog-post
+---
+
+Event Store 5.0.0 is out! It is available for the following operating systems:
+
+- Windows
+- Ubuntu 18.04
+- Ubuntu 16.04
+- Ubuntu 14.04
+- macOS 10.9+
+- CentOS 7 (Commercial version)
+- Amazon Linux AMI (Commercial version)
+
+It's been a few long months since our last stable release of EventStore as we've invested time in increasing the stability of the server, providing higher quality builds and improving our user and developer experience.
```

Comment:
@shaan1337 

```suggestion
It's been some months since our last stable release of EventStore as we've invested time in increasing the stability of the server, providing higher quality builds and improving our user and developer experience.
```